app/components/elements/modal/index.js

```diff
@@ -4,37 +4,21 @@ import {some, diffObj} from 'wasmuth'
 import {dispatch, watchPath} from '/store'
 import {closeModal} from './actions'
 
-watchPath(['modals'], (modals, oldModals = {}) => {
```

Comment:
Why get rid of this? That css class prevents the site from being scrolled when a modal is open

Reply:
because I didn't know why it was there 🙈